app/js/lib/foxbox.js

```diff
@@ -327,6 +326,25 @@ export default class Foxbox extends Service {
     });
   }
 
+  performServiceCommand(id, command, method = 'GET', state = undefined) {
```

Comment:
I want the lib to be easy to understand by regular developers. The long-term of the lib is to be independent from the app and live in its own repo so that 3rd party dev can reuse it.
If they have to look into the API of every single service they want to use, that defeats the point of having a lib in the first place.

I understand that adding many many specific API is cumbersome and we need a better approach.
How about an object oriented approach? You get an instance of `IpCamera` object from the `foxbox.getServices()` and this object has specific API like `.getSnapshot()` or `.getVideoStream()`. This way we don't have to pass service IDs, just to keep a reference to objects.
What do you think?